OpenRA.Mods.Common/Traits/BotModules/SupportPowerBotModule.cs

```diff
@@ -130,7 +130,7 @@ void IBotTick.BotTick(IBot bot)
 			var powerDecision = powerDecisions[readyPower.Info.OrderName];
 			if (powerDecision == null)
 			{
-				AIUtils.BotDebug("Bot Bug: FindAttackLocationToSupportPower, couldn't find powerDecision for {0}", readyPower.Info.OrderName);
+				AIUtils.BotDebug("FindAttackLocationToSupportPower, {0} couldn't find powerDecision for {1}", player.PlayerName, readyPower.Info.OrderName);
```

Comment:
Is it necessary to prefix this and below with `FindAttackLocationToSupportPower`? When we're already at unifying the prefixes, it might make sense to remove this too so everything starts with the player name.

Reply:
Removed those as well. It seems the method names have since been refactored.